### PR TITLE
cpu quota support (#560)

### PR DESCRIPTION
**Pull request by nxmatic**
just added period and quota parameters to container configuration

### CODE REVIEW

File: src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java [hunk 1]
```diff
@@ -565,11 +598,12 @@ public CreateContainerCmd fillContainerConfig(CreateContainerCmd containerConfig
             final Long shmSizeInByte = Long.valueOf(shmSize * 1024L * 1024L);
             containerConfig.getHostConfig().withShmSize(shmSizeInByte);
         }
-
+
```
Comment (pjdarton) [inline src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java]: Please avoid making unnecessary whitespace changes - keep the code changes as tight as possible.

**Comment by steinerthomas** [comment]
What's the status of this pull request?
Can you also add "--cpus=<value>" support for Docker 1.13 and higher?

**Comment by pjdarton** [comment]
Changes were requested to this PR; these have not been done.

If you want to push this forwards, create your own PR based on this one, resolving the changes that were requested (i.e. keep the code changes to a minimum), then make sure the CI build passes.

**Comment by XQDD** [comment]
So, What the progress of this PR?
https://docs.docker.com/engine/reference/commandline/update/

**Comment by pjdarton** [comment]
As far as I am aware, this PR's owner has not progressed this PR.

If you want this functionality, I'd suggest that you create your own PR; that way you'll be able to resolve any review issues raised and we can get it merged in a more timely fashion (so that it doesn't end up with merge issues).

**Comment by pjdarton** [comment]
Closing due to inactivity - seems to have been abandoned in an incomplete state.
